### PR TITLE
update for INTLY-1032

### PR DESCRIPTION
**Pull request by pwright**
adding instead of publishing

updated text around WALKTHROUGH_LOCATIONS

### CODE REVIEW

**Review by odra — APPROVED** [review]

File: walkthroughs/publishing-walkthroughs/walkthrough.adoc [hunk 1]
```diff
@@ -62,28 +66,24 @@ You have the URL one or more repositories with the exact content you want to pub
...
-. Select *Web App* in the resources drop down.
+. Choose *Web App* from the *Other Resources* menu. 
+The tutorial-web-app-operator entry is displayed.
 
-. Click *Actions* and select *Edit YAML* in the resulting drop-down menu.
+. Choose *Edit YAML* from the *Actions* menu.
+An editor is displayed.
 
-. Find the *Spec* > *Template* > *Parameters* entry in this YAML
-. Add (or edit, if it already exists) a parameter:
```
Comment (tremes) [inline walkthroughs/publishing-walkthroughs/walkthrough.adoc:86]: So are we sure the env var will always be there? I think I've seen the case when it wasn't available.

Comment (odra) [reply walkthroughs/publishing-walkthroughs/walkthrough.adoc:86]: It won't at first but I thought it was intuitive enough to add it by saying "edit"

Comment (tremes) [reply walkthroughs/publishing-walkthroughs/walkthrough.adoc:86]: It doesn't seem to me intuitive TBH. We are saying `Edit the `WALKTHROUGH_LOCATIONS` parameter` but the parameter doesn't exist right? But yeah it's likely just minor.

Comment (pwright) [reply walkthroughs/publishing-walkthroughs/walkthrough.adoc:86]: +1 @tremes  I've changed it to `Create or edit`

**Pull request merged**